client/apps/edit/components/content/section_list/index.jsx

```diff
@@ -24,8 +25,17 @@ export class SectionList extends Component {
   }
 
   onDragEnd = (newSections) => {
-    const { resetSectionsAction } = this.props
+    const {
+      article: { layout },
+      logErrorAction,
+      resetSectionsAction
+    } = this.props
 
+    if (newSections[0].type === 'social_embed' && layout === 'news') {
+      return logErrorAction({
+        message: 'Embeds are not allowed in the first section.'
+      })
+    }
```

Comment:
This is slick 👍

Reply:
One thing to note is that @l2succes made a new error modal during the lockout project, will make a new issue to convert it to use with general client errors as well. (The existing logError uses a flash error)